modular_skyrat/modules/customization/modules/client/preferences.dm

```diff
@@ -3282,7 +3282,11 @@ GLOBAL_LIST_INIT(food, list(
 		all_quirks = list()
 
 /datum/preferences/proc/get_linguistic_points()
-	var/points = LINGUISTIC_POINTS_DEFAULT
+	var/points
+	if("Linguist" in all_quirks)
+		points = LINGUISTIC_POINTS_LINGUIST
+	else
+		points = LINGUISTIC_POINTS_DEFAULT
```

Comment:
This would be a cleaner way of doing it, and should actually work, because I actually doubt the original code would work because the "L" is capitalized when it shouldn't. Either way, you had a define, better to use it.
```suggestion
	points = LINGUISTIC_POINTS_LINGUIST
	points = TRAIT_LINGUIST in all_quirks ? points + LINGUISTIC_POINTS_LINGUIST : points
```